class 4

```diff
@@ -0,0 +1,227 @@
+/*console.log('welcome to class 4');
+
+
+console.log('first');
+console.log('second');
+setTimeout(() => {                                   // asynchronous function 
+    console.log('third');
+
+    
+}, 0);
+
+console.log('fourth');                                // out -> first second fourth third  sequentially  hoi nai 
```

Comment:
Try to write comment in **English** instead of **Banglish**. As this repo is a public repo and anyone can see your code. If anyone want to learn something from your code, then it is better to write comment in English. Though, I know you are commenting this way for self learning but I would like to suggest you to write comment in English, also it is a good habitual fact indeed.